CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 
 ## 0.3.6:
 
-#### Released: Oct, 1st, 2016
+#### Released: October 1st, 2016
 #### [View](https://github.com/CactusDev/CactusBot/releases/tag/v0.3.6) 
```

Comment:
Rather than having `[View]` under each version, could we just link the version name?

```
## [v0.3.6](https://...)
```